provision/initramfs/capabilities/setup-filesystems/20-filesystems

```diff
@@ -169,28 +229,6 @@ update_fstab_and_mount() {
         SELECT_DISK=""
       fi
       SELECT_DISK=$(echo "${FSCMD}" | cut -f 2 -d' ')
-
-      SELECT_RETRY=0
-
-      if [ -z "${SELECT_DISK##*/dev/*}" ]; then
```

Comment:
We recently added this logic in to deal with disks not being ready. What problems did you run into with it?

Reply:
I originaly worked on master base, so I did not notice this change. I will look into it.

Reply:
I have moved it to separate function. This way, we can use it to waif for partition in case `udevadm` is not available.